### PR TITLE
Add a backwards compatible S3 Host override via ENV

### PR DESCRIPTION
**Pull request by DemonusPC**

### CODE REVIEW

File: lib/alephant/storage.rb [hunk 1]
```diff
@@ -95,9 +95,18 @@ def add_custom_meta(object)
...
+    def override_host_path?
+      ENV['AWS_S3_HOST_OVERRIDE'] == 'true'
+    end
+
+
     def client
       options = {}
       options[:endpoint] = ENV['AWS_S3_ENDPOINT'] if ENV['AWS_S3_ENDPOINT']
+      if override_host_path?
+        options[:disable_host_prefix_injection] = true
```
Comment (owenj62) [inline lib/alephant/storage.rb:107]: Probably want a test around this just checking the right options are set for each path.

Comment (DemonusPC) [reply lib/alephant/storage.rb:107]: tested manually but good point

Comment (DemonusPC) [reply lib/alephant/storage.rb:107]: done

**Review by owenj62 — APPROVED** [review]
LGTM 👍

**Pull request merged**